dm/dm/worker/subtask.go

```diff
@@ -151,6 +151,14 @@ func NewSubTaskWithStage(cfg *config.SubTaskConfig, stage pb.Stage, etcdClient *
 
 // initUnits initializes the sub task processing units.
 func (st *SubTask) initUnits() error {
+	// NOTE: because lighting not support init tls with raw certs bytes, we write the certs data to a file.
```

Comment:
```suggestion
	// NOTE: because lightning not support init tls with raw certs bytes, we write the certs data to a file.
```

and rest place

Reply:
fixed in https://github.com/pingcap/ticdc/pull/3260/commits/e35473de552e3413d1b8d69697d522d1f8bb62b6